easybuild/easyconfigs/m/minizip/minizip-2.10.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'minizip'
+version = '2.10.0'
+
+homepage = 'https://github.com/nmoinvaz/minizip'
+description = """minizip is a zip manipulation library written in C that is supported on Windows, macOS, and Linux."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+# on github the file resides at: https://github.com/nmoinvaz/minizip/archive/2.10.0.tar.gz
+github_account = 'nmoinvaz'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
+]
+
+dependencies = []
+
+# The sanity test MUST be tuned before going production and submitting your contribution to upstream git repositories
+sanity_check_paths = {
+    'files': ["lib64/libminizip.a"],
```

Comment:
It would be good to get the shared library as well. e.g. `configopts = ['-DBUILD_SHARED_LIBS=OFF', '-DBUILD_SHARED_LIBS=ON']`